.github/workflows/release.yml

```diff
@@ -54,4 +54,11 @@ jobs:
         with:
           repository-name: "${{ github.event.repository.name }}"
           package-path: ${{ steps.local_repo.outputs.dir }}
-          release-name: "${{ github.event.inputs.version}}"
\ No newline at end of file
+          release-name: "${{ github.event.inputs.version}}"
+      - name: Notify failures on Slack
```

Comment:
maybe it would make sense to take the trigger of this GHA into consideration as well and add an additional notification on cirrus? if you think this is enough it makes sense to me as well